src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -431,6 +432,14 @@ public Index GetIndex(string name)
 
                 return instance;
             }
+            catch (TimeoutException toe)
+            {
+                throw new IndexCreationException($"Failed to create auto index: {definition.Name}, the cluster is probably down.", toe);
```

Comment:
We also want to give the user the current state of the cluster.

Reply:
@ayende do you mean, proving the cluster topology?

Reply:
You need to provide the cluster _status_, not the topology.
When error reporting, we don't go outside our local state. We need:
* Current node state (follower, candidate, leader)
* Last status change reason (and when it happened).